index.html

```diff
@@ -32,17 +32,17 @@ <h1>The V Programming Language</h1>
         <span class='name'>Linux</span>
-        <span class='size'>6.6MB</span>
```

Comment:
I left these nodes empty intentionally to avoid confusing.

We could also add a note this text content is generated, e.g.:
```html
<span class='size'><!-- Generated by script --></span>
```